packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/check-for-auth-migration.ts

```diff
@@ -10,10 +11,15 @@ export const checkAuthResourceMigration = async (context: $TSContext, authName:
   if (!imported) {
     const cliState = new AuthInputState(authName);
     if (!cliState.cliInputFileExists()) {
-      printer.debug('Cli-inputs.json doesnt exist');
+      printer.debug("cli-inputs.json doesn't exist");
       // put spinner here
       const headlessMigrate = context.input.options?.yes || context.input.options?.forcePush || context.input.options?.headless;
-      if (headlessMigrate || (await prompter.yesOrNo(`Do you want to migrate this ${authName} to support overrides?`, true))) {
+      const docsLink = 'https://docs.amplify.aws/cli/migration/overrides';
+      const migrateResourceMessage =
```

Comment:
We also have a similar message for storage - can we make this change there as well?

@akshbhu are there any migrations tied to the root-stack?

Reply:
@kaustavghosh06 where is the message being printed for storage?